src/Tizen.NUI/src/internal/Interop/Interop.DragAndDrop.cs

```diff
@@ -41,6 +41,12 @@ internal enum DragType
             [global::System.Runtime.InteropServices.DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_DragAndDrop_RemoveListener")]
             public static extern bool RemoveListener(global::System.Runtime.InteropServices.HandleRef dragAndDrop, global::System.Runtime.InteropServices.HandleRef targetView, global::System.Runtime.InteropServices.HandleRef callback);
 
+            [global::System.Runtime.InteropServices.DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_DragAndDrop_Window_AddListener")]
+            public static extern bool WindowAddListener(global::System.Runtime.InteropServices.HandleRef dragAndDrop, global::System.Runtime.InteropServices.HandleRef targetWindow, global::System.Runtime.InteropServices.HandleRef callback);
```

Comment:
For the interop function of return type `bool`, it needs to specify that it uses 1byte for `bool`.
(It's 4 bytes by default)
[MS Guide](https://learn.microsoft.com/en-us/dotnet/api/system.boolean?view=net-7.0#booleans-and-interop))
```C#
[return: MarshalAs(UnmanagedType.U1)]
```

Reply:
I have modified it according to your comment :)